sdk/formrecognizer/ci.yml

```diff
@@ -0,0 +1,42 @@
+# DO NOT EDIT THIS FILE
+# This file is generated automatically and any changes will be lost.
```

Comment:
Hmmmmmm...

Reply:
I was quite confused about this.

Reply:
LOL... yeah lets call it a deterrent to keep people from playing with these files. We do have a goal to generate these at some point also but just haven't gotten to it yet.